extensions/spring-rabbitmq/runtime/src/main/doc/limitations.adoc

```diff
@@ -0,0 +1,5 @@
+You can use component without any special configuration in the JVM mode.
+
+In native mode you have to add compiler flag `-H:+InlineBeforeAnalysis`,
+to allow removal in static methods (see https://github.com/oracle/graal/issues/2594[graal issue]).
+Component fails without this flag in the native mode.
```

Comment:
```suggestion
In native mode you need to add 

[source,shell]
----
quarkus.native.additional-build-args = -H:+InlineBeforeAnalysis 
----

to your `application.properties`. This is to allow inlining of some static methods that would otherwise cause build failures (see this https://github.com/oracle/graal/issues/2594[GraalVM issue]).
```

Reply:
Thanks, fixed.